### PR TITLE
CPM-430 Add Surtitle on dropdown

### PR DESCRIPTION
**Pull request by cemorin**
Add Surtitle component in order to add ref entities inside table attribute

**Definition Of Done (for Core Developer only)**

- [x] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: front-packages/akeneo-design-system/src/components/Dropdown/Item/Item.tsx [hunk 1]
```diff
@@ -6,26 +6,40 @@ import {Checkbox} from '../../../components/Checkbox/Checkbox';
...
-  line-height: 34px;
 `;
 
-const ItemContainer = styled.div<{tall: boolean; disabled: boolean; isActive: boolean} & AkeneoThemedProps>`
+const sizeMap = {
+  default: 34,
+  big: 44,
+  bigger: 64,
+};
+const ItemContainer = styled.div<
```
Comment (pierallard) [inline front-packages/akeneo-design-system/src/components/Dropdown/Item/Item.tsx:23]: Add a line before plz :grimacing:

**Review by pierallard — APPROVED** [review]
GTM once fixed

File: front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx [hunk 2]
```diff
@@ -382,3 +384,55 @@ For accessibility reasons, we must put the full text in an HTML title attribute.
...
+                    <Image src="https://picsum.photos/seed/akenep/200/140" alt="The alt" />
+                    <Surtitle label="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.">
+                      Item 2
+                    </Surtitle>
+                    <Badge level="danger">50%</Badge>
+                    <IconButton icon={<LinkIcon/>} ghost="borderless" level="tertiary"/>
+                  </Dropdown.Item>
+                  <Dropdown.Item title="Item 3">
+                    <Image src="https://picsum.photos/seed/akeneq/200/140" alt="The alt" />
+                    <Surtitle label="Item 3 Annotation">
```
Comment (juliensnz) [inline front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx:424]: Will the Surtitle used elsewhere than in the Dropdown?
If it's only used in the dropdown, what do you think about putting it in the Dropdown folder and only expose it as `<Dropdown.Surtitle>` ?

**Review by juliensnz — APPROVED** [review]
Looks good to me. The only weird thing IMHO is that Surtitle is exposed as a standalone component while it's only used in the Dropdown.

File: front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx (hunk 2 above)
Comment (pierallard) [reply front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx:424]: @juliensnz Seen with Stéphane, it's only for Dropdown
We added it as Dropdown.Surtitle but we forget in this place, yes, nice catch.

Comment (cemorin) [reply front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx]: It's fixed, it's now only used as <DropDown.Surtitle>

**Pull request merged**